### DISCUSSION TITLE
Configuring duplicate_action

### DISCUSSION THREAD

**Discussion by nsteins** [beetbox/beets#4763] [Q&A] [answered]
I am setting up some automated imports on a collection of music downloaded from bandcamp, and there are occasional duplicates because of the way the downloader is configured. I would like to skip imports of duplicates by setting the `duplicate_action` tag, but is there a way to configure how duplicates are determined? The default behavior of `beet duplicates` relies on musicbrainz ids which does not work here, since most tracks are not on musicbrainz. I have been able to effectively find duplicates with `beet duplicates -k title -k albumartist -k album` however. Is there a way to make this the default method of determining duplicates for the `duplicate_action` tag?

**Comment by sampsyo** [comment, marked as answer]
Duplicate detection in the importer is, confusingly, not related at all to the `duplicates` plugin (and the `beet duplicates` command it provides). While it's only in git, #4199 added a configuration option that controls which metadata is used to detect duplicates in the importer.

**Reply by nsteins** [reply]
Thank you, this is very helpful. Curious, prior to the addition of the configuration option, what keys were used as the default for detecting duplicates?

**Reply by sampsyo** [reply]
The same as the current defaults: `albumartist` and `album` for albums and `artist` and `title` for items.
